browser/resources/settings/brave_overrides/search_engines_page.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import {RegisterPolymerTemplateModifications} from 'chrome://resources/brave/polymer_overriding.js'
+
+RegisterPolymerTemplateModifications({
```

Comment:
Hey, the result here looks great, but do you think you could try using a Polymer Style override instead?

It should let you inject CSS into the component, and hopefully we can avoid needing to directly set the `style` attribute.

```ts
RegisterStyleOverride(
  'settings-search-engines-page',
  html`
    <style>
      #addSearchEngine {
        float: right;
        margin: 0 5% 0 0;
      }
    </style>
  ` )
```
(this is just me guessing)

You might still need the RegisterTemplateModifications to move the element, but you might be able to do it with some clever `display:flex; flex-direction: column-reverse` or something. All good if not!

Reply:
I moved style changes to RegisterStyleOverride method.
Regarding using `display:flex` to right-align the button. Post moving the button under 
```    
<settings-search-engines-list
        hidden="[[!matchingOtherEngines_.length]]"
        engines="[[matchingOtherEngines_]]"
        scroll-target="[[subpageScrollTarget]]"
        show-query-url collapse-list
        expand-list-text="$i18n{searchEnginesAdditionalInactiveSites}">
</settings-search-engines-list>
```
there is no parent div containing button and `settings-search-engines-list`. I think it may require us to introduce a div as a parent of `settings-search-engines-list` and `cr-button` element, and then play with flex. Should we go this route? 
Also, AFAIK, SearchEnginesPage is a desktop feature only so the responsiveness which flex provides isn't at stake. I  tested by shrinking and expanding the browser and the button didn't seem awry. 
Correct me if I've understood it wrong.

Reply:
All good to keep the moving code how it is!